src/sampling/sampler_npi.py

```diff
@@ -30,7 +30,7 @@ def run(self):
         print("computing kappa for base_r0=" + str(self.base_r0))
         number_of_samples = self.sim_obj.n_samples
         lhs_table = self._get_lhs_table(number_of_samples=number_of_samples,
-                                        kappa=kappa)
+                                        kappa=0)
```

Comment:
For kappa, please create an additional argument for the constructor of SamplerNPI (`is_kappa_applied`), which is set to `True` by default. If it is `True`, then kappa calculation in LOC 29 is executed and the result is passed, otherwise (`else`) `kappa = None` - in this case, no kappa approach will be used.